### DISCUSSION TITLE
Can i place one css class inside another?

### DISCUSSION THREAD

**Discussion by Juanidevelp** [community/community#136788] [Programming Help] [answered]
### Body

like this:

.clase1 {
  color: red;

  .clase2 {
    color: blue;
  }
}

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by isma-is-dev** [comment, marked as answer]
No, it's a feature of the SCSS preprocessor. You can use it in https://sass-lang.com
